src/pages/dataTableUtilities/getColumns.js

```diff
@@ -41,11 +41,20 @@ const PAGE_COLUMN_WIDTHS = {
   [ROUTES.SUPPLIER_REQUISITIONS]: [1.5, 2, 1, 1, 1, 1],
   [ROUTES.PRESCRIPTION]: [2, 4, 2, 2, 1],
   [ROUTES.PRESCRIPTIONS]: [1.5, 2.5, 2, 1.5, 3, 1],
+  fridges: [1, 3],
+  sensors: [1, 3, 2, 2],
```

Comment:
happy to get ideas here - technically not a route/modal/tab.. maybe I should use tab? Or data set? I dunno :(

Reply:
Hmm, maybe slightly breaking the pattern, but could get away with `VACCINES`, e.g. `VACCINES.FRIDGES`?

Reply:
I think something more generic is a bit nicer? `DATA_SET.FRIDGES` or something? Seems to be a pattern we're using more and more